test-dssspell.sh

```diff
@@ -3,8 +3,8 @@ set -e
 
 [[ "$ETH_RPC_URL" && "$(seth chain)" == "ethlive"  ]] || { echo "Please set a mainnet ETH_RPC_URL"; exit 1;  }
 
-export DAPP_BUILD_OPTIMIZE=1
-export DAPP_BUILD_OPTIMIZE_RUNS=1
+#export DAPP_BUILD_OPTIMIZE=1
+#export DAPP_BUILD_OPTIMIZE_RUNS=1
```

Comment:
nit: commented opts in tests

Reply:
Well it is not actually a `nit`. This needs to be uncommented.

Reply:
yep, you're right. 
we could `cherry nit-pick` into something like `test-noop.sh` as @brianmcmichael suggested in chat to be used for testing during development via `make noop`.

Reply:
oops, yes would prefer a better way to do this so I don't forgot stuff like this